.github/workflows/Unit-Test.yml

```diff
@@ -130,25 +130,27 @@ jobs:
             case $TARGET_ARCH in
               x86_64)
                 # Export the x86_64 container depending on whether this PR updates packages with appropriate minimum glibc.
-                if [[ $GLIBC_232_COMPATIBLE_PACKAGES ]]; then
-                    echo "CONTAINER=nocturne-x86_64.m97" >> "$GITHUB_ENV"
-                elif [[ $GLIBC_237_COMPATIBLE_PACKAGES ]]; then
-                    echo "CONTAINER=hatch-x86_64.m135" >> "$GITHUB_ENV"
-                else
-                    echo "CONTAINER=nocturne-x86_64.m90" >> "$GITHUB_ENV"
-                fi
+                # if [[ $GLIBC_232_COMPATIBLE_PACKAGES ]]; then
+                #     echo "CONTAINER=nocturne-x86_64.m97" >> "$GITHUB_ENV"
```

Comment:
@satmandu We don't need to care about the `glibc` version of the container anymore, as I have extracted some glibc libraries from the oldest available ChromeOS images (link with those libraries if we want binaries also works on system's `glibc`, see [chromebrew/crew-package-glibc](https://github.com/chromebrew/crew-package-glibc/tree/main/prebuilt)). This also means we can now use the latest version of ChromeOS for building things

Reply:
That's fine... It was just using the M97 containers for tests, which was a waste of time...

Reply:
The older chromeos based containers are smaller, so probably load faster?

Reply:
Maybe try stripping out the Android subsystem (`/opt/google/vms/android`) and Chrome (`/opt/google/chrome`) when building a new container image next time? That would save a ton of space for us.

Reply:
This is in the container image script being used starting tonight ...